packages/shopware-6-client/src/services/checkoutService.ts

```diff
@@ -39,3 +47,70 @@ export async function handlePayment(
 
   return resp.data;
 }
+
+/**
+ * Get order details
+ *
+ * @throws ClientApiError
+ * @beta
+ */
+export async function getOrderDetails(
+  orderId: string,
+  params?: Object,
```

Comment:
shouldn't it be searchParams? We need a type here

Reply:
also, what would the additional params be? If we're fetching by id we don't need to pass any additional params I guess